src/CSharp/CodeCracker/Style/PropertyPrivateSetAnalyzer.cs

```diff
@@ -39,7 +39,7 @@ private static void AnalyzeClass(SyntaxNodeAnalysisContext context)
             var invocationExpression = (PropertyDeclarationSyntax)context.Node;
             var semanticModel = context.SemanticModel;
 
-            if (invocationExpression.AccessorList.Accessors.Count == 1) return;
+            if (invocationExpression.AccessorList == null || invocationExpression.AccessorList.Accessors.Count == 1) return;
```

Comment:
Sure, but the if would have to change to:

    (invocationExpression.AccessorList?.Accessors.Count ?? 1) == 1

Should I change it?